packages/itwinui-react/src/core/Tree/Tree.tsx

```diff
@@ -380,11 +403,29 @@ const VirtualizedTree = React.forwardRef(
     }: VirtualizedTreeProps<T>,
     ref: React.ForwardedRef<HTMLUListElement>,
   ) => {
-    const { outerProps, innerProps, visibleChildren } = useVirtualization({
-      itemsLength: flatNodesList.length,
-      itemRenderer: itemRenderer,
+    const parentRef = React.useRef<HTMLDivElement | null>(null);
+
+    const virtualizer = useVirtualScroll(
+      flatNodesList.length,
+      () => parentRef.current,
+      () => 39, //Set to 39px since that is the height of a treeNode with a sub label with the default font size.
       scrollToIndex,
-    });
+    );
```

Comment:
Yeah, I think this makes sense. Added a parameter to the `useVirtualScroll` hook for this and wrapped the parameters for the hook in an object, so that the order parameters are passed into the hook won't matter anymore. Added a callback for getting the `nodeId` and passed into `useVirtualScroll` in this [commit](https://github.com/iTwin/iTwinUI/pull/2091/commits/2dcf4c7b94ba24be715aac59646e0359e8b3599d).